content/data-management-plan.md

```diff
@@ -0,0 +1,28 @@
+# Data Management Plan (DMP)
+
+Data management plans help structure data handling throughout a research project. Simply put, data management plans ask the following guiding questions:
+  * Where can collaborators find the data?
+  * Is the data accessible to others?
+  * Is the data interoperable?
+  * Is the data reusable?
+
+Walking through these questions for a research project can help identify potential problems. For example, it might become clear that only one person has access to the data an entire project depends on. Such a situation can easily result in data loss (a project with a [bus factor](https://en.wikipedia.org/wiki/Bus_factor) of one). If that person has extensive back-up plans to prevent data loss, the problem is mitigated somewhat.
+
+Funding agencies increasingly require data management plans, because they heavily invest in research projects and want to maximize their value for money. But data they are super helpful to you as an individual. 
```

Comment:
"But data they are super helpful to you as an individual."

Reword?

Reply:
Hahahaha yea that is a shoddy sentence. Incorporated in next commit!